modules/apps_api/Gemfile.lock

```diff
@@ -0,0 +1,111 @@
+PATH
```

Comment:
I don't think we want to commit a Gemfile.lock for a module

Reply:
This file was unintentionally added. Done!